raiden-dapp/src/components/account/Withdrawal.vue

```diff
@@ -0,0 +1,203 @@
+            <amount-display
+              inline
+              :amount="withdraw.balance"
+              :token="withdraw"
+            />
+          </i18n>
+          <div
+            v-if="parseFloat(raidenAccountBalance) === 0"
+            class="error--text mt-2"
+          >
+            {{ $t('withdrawal.dialog.no-eth') }}
+          </div>
+        </div>
+        <div v-else class="mt-4">
+          <v-progress-circular
+            :size="86"
+            :width="4"
+            color="primary"
+            indeterminate
+          ></v-progress-circular>
+          <div class="mt-4">{{ $t('withdrawal.dialog.progress') }}</div>
+        </div>
+      </v-card-text>
+      <v-card-actions v-if="!withdrawing">
+        <action-button
+          class="withdrawal-dialog__action"
+          :enabled="parseFloat(raidenAccountBalance) > 0"
+          :text="$t('withdrawal.dialog.button')"
+          @click="withdrawTokens()"
+        >
+        </action-button>
+      </v-card-actions>
+    </raiden-dialog>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator';
+import { mapState } from 'vuex';
+import { Tokens } from '@/types';
+import { Token } from '@/model/types';
+import AddressDisplay from '@/components/AddressDisplay.vue';
+import BlockieMixin from '@/mixins/blockie-mixin';
+import AmountDisplay from '@/components/AmountDisplay.vue';
+import ActionButton from '@/components/ActionButton.vue';
+import RaidenDialog from '@/components/dialogs/RaidenDialog.vue';
+
+@Component({
+  components: { ActionButton, RaidenDialog, AddressDisplay, AmountDisplay },
+  computed: {
+    ...mapState(['tokens', 'raidenAccountBalance'])
+  }
+})
+export default class Withdrawal extends Mixins(BlockieMixin) {
+  tokens!: Tokens;
+  raidenAccountBalance!: string;
+  balances: Token[] = [];
+  loading: boolean = true;
+  withdrawing: boolean = false;
+  withdraw: Token | null = null;
+
+  async mounted() {
+    this.balances = await this.$raiden.getRaidenAccountBalances();
+    this.loading = false;
+  }
+
+  async withdrawTokens() {
+    if (!this.withdraw || !this.withdraw.balance) {
+      return;
+    }
+    try {
+      this.withdrawing = true;
+      const { address, balance } = this.withdraw;
+      await this.$raiden.transferOnChainTokens(address, balance);
+      this.balances = this.balances.filter(token => token.address !== address);
+      this.withdraw = null;
+    } catch (e) {
+    } finally {
+      this.withdrawing = false;
+    }
+  }
+}
+</script>
+
+<style scoped lang="scss">
+.withdrawal {
+  &__description {
+    margin-top: 30px;
+    text-align: center;
+  }
+
+  &__loading,
+  &__empty {
+    min-height: 490px;
+  }
+
+  &__tokens {
+    margin-top: 60px;
+    background-color: transparent;
+
+    &__balance {
+      margin-right: 12px;
+    }
+
+    &__address {
+      ::v-deep {
+        .address__label {
+          color: #ffffffb3 !important;
```

Comment:
```suggestion
          color: rgba($color-white, 0.7) !important;
```

But would also need `@import '@/scss/colors';`

Reply:
ok let me check that and also fix the linting issue that I didn't see locally :disappointed: